pages/authentication/logout.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../../css/main.css" />
+    <script src="https://kit.fontawesome.com/8831240330.js" crossorigin="anonymous"></script>
+    <title>Logout</title>
+</head>
+
+<body>
+    <nav class="nav">
+        <a href="../../index.html">
+            <h1>Brand</h1>
+        </a>
+        <div class="search-box">
+            <input type="search" name="search" placeholder="search ..." />
+            <i class="fas fa-search search-btn"></i>
+        </div>
+        <div class="open-menu"><i class="fas fa-bars"></i></div>
+        <ul class="main-menu text-s fw-600">
+            <li><a href="#">Men</a></li>
+            <li><a href="#">Women</a></li>
+            <li><a href="#">Accessories</a></li>
+            <li>
+                <a href="../authentication/login.html">
+                    <button class="btn btn-primary login-btn">Login</button>
+                </a>
+            </li>
+            <li>
+                <a href=""><i class="fas fa-heart"></i></a>
+            </li>
+            <li>
+                <a href=""><i class="fas fa-cart-plus"></i></a>
+            </li>
+            <div class="close-menu"><i class="fa fa-times"></i></div>
+            <span class="socials">
+          <li>
+            <a href=""><i class="fab fa-instagram"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fab fa-twitter"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fab fa-github"></i></a>
+          </li>
+        </span>
+        </ul>
+    </nav>
+
+    <section class="auth-container">
+        <div class="auth-content">
+            <div class="text-center"><i class="fa fa-check-circle fa-2x"></i></div>
+            <p>You have been logged out. Click below to login</p>
+            <a href="../authentication/login.html"><button class="btn btn-primary text-s">Login</button></a>
+        </div>
+    </section>
+
+    <footer class="footer">
+        <h3>Keep in touch</h3>
+        <ul>
+            <a href="">
+                <li><i class="fab fa-instagram"></i></li>
+            </a>
+            <a href="">
+                <li><i class="fab fa-twitter"></i></li>
+            </a>
+            <a href="">
+                <li><i class="fab fa-github"></i></li>
+            </a>
+        </ul>
```

Comment:
Since you have used <ul>, next you should have <li> and then <a> as <li> are placed under <ul>.